### DISCUSSION TITLE
Rajouter les messages de reconnexion

### DISCUSSION THREAD

**Discussion by LeGeek01** [PapillonApp/papillon-v5#84] [General]
### Description du problème

Je ne trouve pas utilise de supprimer les messages de reconnexion comme Vincent vient de le faire. Cela permet une information sur l'état de la connexion des serveurs, et peut expliquer à l'utilisateur pourquoi la requête prend du temps à être traitée.

### Description de l'amélioration

Remettre donc les messages comme ils étaient au départ.

**Comment by ecnivtwelve** [comment]
Je les ai supprimé car effectivement ils indiquent l'état de connexion à l'utilisateur, mais cela au prix d'une notification assez perturbante et récurrente. En attendant de trouver une meilleure alternative, je préfère remplacer ces notifications par des messages d'erreur ou de chargement prolongé dans les onglets directement pour conserver une UX correcte sur les chargements sans pour autant constamment déranger l'utilisateur

**Comment by TheTiger72** [comment]
Sinon il faut modifier le délai avant une reconnexion

**Reply by LeGeek01** [reply]
Ça c'est pas trop possible, la session expire au bout de 5 minutes d'inactivité ou plus je sais plus

**Comment by LeGeek01** [comment]
Je ne trouve pas que ceci dérange l'utilisateur, le cas contraire il y aurait plus d'issues sur le github ou de plaintes sur le discord, chose qui n'est pas le cas actuellement

**Reply by ecnivtwelve** [reply]
Je pense pas que quelqu'un irait jusqu'a faire une issue pour ça, mais ca reste un élement qui perturbe **inutilement** l'expérience utilisateur. Vu que la reconnexion est automatique, il n'y a plus besoin d'informer sur son état.
